src/app/server/Server.h

```diff
@@ -146,6 +142,10 @@ struct ServerInitParams
     // Operational certificate store with access to the operational certs in persisted storage:
     // must not be null at timne of Server::Init().
     Credentials::OperationalCertificateStore * opCertStore = nullptr;
+    // Required, if not provided, the Server::Init() WILL fail. The default reportScheduler implementation will be used when using
+    // CommonCaseDeviceServerInitParams, but the injection of a custom scheduler requires using the ServerInitParams base class to
```

Comment:
You shouldn't even mention CommonCaseDeviceServerInitParams here.  People should not be using it.